src/Misc/dotnet-install.sh

```diff
@@ -144,7 +144,7 @@ get_linux_platform_name() {
     else
         if [ -e /etc/os-release ]; then
             . /etc/os-release
-            echo "$ID.$VERSION_ID"
+            echo "$ID${VERSION_ID:+.${VERSION_ID}}"
```

Comment:
Same here.

Reply:
This file is from outside of this repo and just we needed to download it to the agent repo